### ISSUE TITLE
Display canvas word counts

### ISSUE THREAD

**Issue by LiamSwayne** [isaaclyman/novel-word-count-obsidian#80]
**Problem**

Canvas word counts are not displayed.

**Idea**

Sum the word counts of the text-block nodes in canvas, excluding embedded notes and embedded canvases. These text blocks can then be parsed like regular notes. This would give an accurate count of the words exclusive to the canvas and would avoid counting embedded notes' word counts multiple times.

**Comment by isaaclyman** [comment]
Canvas files are stored in JSON format as follows:

```
{
	"nodes":[
		{"id":"beba9a4110250da2","x":46,"y":-118,"width":400,"height":400,"type":"file","file":"Languages/Japanese (325w).md"},
		{"id":"1376822f2272ea36","x":-280,"y":-240,"width":250,"height":60,"type":"text","text":"Custom text has five words"}
	],
	"edges":[]
}
```

It should be possible to parse them and get a word count from the text nodes, but I'll have to do some research.

**Comment by isaaclyman** [comment]
Fixed in v3.7.0.

**Comment by LiamSwayne** [comment]
Thanks so much for implementing this, it's really nice to have!
